lib/handbook/addon/docs/visual-style/template.md

```diff
@@ -1,5 +1,98 @@
 # Visual Style
 
-TODO
+## General principles
```

Comment:
I don't know how I feel about copying these over as-is... but I guess it's better to have them here and maybe revise them over time, if it makes sense to.

Reply:
It's a place to start?...

Reply:
Yeah, we should definitely think about ways to make these better, but since we want to be away from osf-style in any case, at least having it in our repo means we can easily make these changes.